validate/validate.go

```diff
@@ -296,14 +302,22 @@ func validateXfam(value []byte, fixed1, fixed0 uint64) error {
 	return nil
 }
 
-func validateTdAttributes(value []byte, fixed1, fixed0 uint64) error {
+func validateTdAttributes(value []byte, fixed1, fixed0 uint64, tdAttributesDebug *bool) error {
 	if len(value) == 0 {
 		return nil
 	}
 	if len(value) != abi.TdAttributesSize {
 		return fmt.Errorf("tdAttributes size is invalid")
 	}
 	tdAttributes := binary.LittleEndian.Uint64(value[:])
```

Comment:
nit: While you are here could you please remove this? Unless I am missing something I'm not seeing the value in creating a new slice here

Reply:
It didn't highlight as I expected, by "remove this" I mean change value[:] -> value